public/app/plugins/panel/nodeGraph/editor/ArcOptionsEditor.tsx

```diff
@@ -44,7 +41,14 @@ export const ArcOptionsEditor = ({ value, onChange, context }: ArcOptionsEditorP
               onChange={(val) => {
                 updateField(i, 'field', val);
               }}
-              item={fieldNamePickerSettings}
+              // @ts-ignore
+              item={{
+                settings: {
+                  filter: (field: Field) => field.name.includes('arc__'),
+                },
+                id: `arc-field-${i}`,
+                name: `arc-field-${i}`,
```

Comment:
do you think it makes sense to add `editor: () => null` and remove `// @ts-ignore`? tradeoff?

Reply:
Definitely, let me test that out! Every other instance of this component just uses settings and a bunch of `any`s which fails the new betterer tests which is why I updated it.

Reply:
Yeah, that worked great!